src/filehasher.js

```diff
@@ -6,10 +7,14 @@ import createHash from 'sha.js';
 import log from 'logger';
 import { lstatPromise } from 'io/utils';
 
+// TODO: This will be removed when hashes are coming
+// from the dispensary module..
```

Comment:
Ever so minor, but that should be one period for a sentence, or three for an ellipsis...